src/bindings/python/src/openvino/frontend/pytorch/patch_model.py

```diff
@@ -82,14 +82,20 @@ def __make_16bit_traceable(model: torch.nn.Module):
      - Replace known list of modules with ModuleExtension.
      - Convert other modules with weights to FP32.
     """
+    def patch_condition(module):
+        supported = [torch.float32, torch.float16, torch.bfloat16]
+        return (hasattr(module, "weight")
+                and getattr(module.weight, "dtype", None) in supported)
+
     extensions = {
         torch.nn.Linear: ModuleExtension(
             torch.nn.Linear, "ov_ext::linear",
```

Comment:
what are `ov_ext::linear` or `ov_ext::embedding`? Where does it come from?

Reply:
It is our internal types to distinguish from regular ops on conversion stage